R/create_hb_test_flags.R

```diff
@@ -11,19 +11,67 @@
 create_hb_test_flags <- function(data, hb_var) {
   data <- data %>%
     dplyr::mutate(
-      NHS_Ayrshire_and_Arran = dplyr::if_else({{ hb_var }} == "S08000015", 1L, 0L),
+      NHS_Ayrshire_and_Arran = dplyr::if_else(
+        {{ hb_var }} == "S08000015",
+        1L,
+        0L
+      ),
       NHS_Borders = dplyr::if_else({{ hb_var }} == "S08000016", 1L, 0L),
```

Comment:
Borders and FV have not changed on here, should these also reshape? I tried to install code grip and run the reshape but it also did not change so maybe it is not an issue?